### DISCUSSION TITLE
VSCodeWorkbench mount failed with "The operation is insecure"

### DISCUSSION THREAD

**Discussion by maciej-pesta** [community/community#29495] [Codespaces]
Hello,
I can't load my Codespaces at CS50 course. When it starts after a while I have below error:

VSCodeWorkbench mount failed with "The operation is insecure"

Could someone help me ? :)

Regards,

**Comment by stepinac** [comment]
Hello, 

I have the same problem. Yesterday it worked fine and this morning I can't log to my codespace.

**Comment by camillaabalzarotti** [comment]
Me too, and I have no idea how to fix it :(

**Comment by Pableil** [comment]
I tried this solution by @softwaredevelop and it works
https://github.com/orgs/community/discussions/29505#discussioncomment-3397080

**Comment by gcsiapn** [comment]
me too

**Comment by MM3603** [comment]
me as well, does anybody know if the servers are down?
